server/clientRoutes.js

```diff
@@ -18,7 +18,10 @@ function clientsRouter(clients, defaultId) {
         `Client config ${id} had no chain set, defaulting to 'bitcoin'`
       );
     clientInfo[client.config.str('id')] = {
-      chain: client.config.str('chain', 'bitcoin')
+      chain: client.config.str('chain', 'bitcoin'),
+      hasNode: !!client.nodeClient,
```

Comment:
What do you think of something like

```
services: {
  node: true,
  chain: true,
  multisig,
}
```
instead of many top level bools

Reply:
Yeah, I like that. Cleaner.